lib/moab/storage_object_validator.rb

```diff
@@ -49,6 +51,7 @@ def self.error_code_to_messages
           NO_SIGNATURE_CATALOG => "Version: %{addl} Missing signatureCatalog.xml",
           NO_MANIFEST_INVENTORY => "Version: %{addl} Missing manifestInventory.xml",
           NO_XML_FILES => "Version: %{addl} Missing all required metadata files",
```

Comment:
is this about data/METADATA or about MANIFESTS ?  Or both?  Does *metadata* have required files?  Are they all XML?

Reply:
NO_XML_FILES is about the metadata directory. I guess instead of saying "Missing all required metadata files" It should say "Missing .xml files and contains other items"? Metadata _does_ _not_ have required files, the directory should only contain files.